File: Python/Exercicios/Ex17.py
# Entrar com o peso, o sexo e a altura de uma determinada pessoa. Após a digitação, exibir se esta pessoa está ou não com seu peso ideal. Fórmula: peso/altura².

p = float(input('Digite o seu peso: '))
a = float(input('Digite a sua altura: '))
sexo = input( 'Digite M para masculino e F para feminino: ')

imc = p/(a * a) 

if sexo == 'f':
    if (imc < 19):
        print(f'Você está abaixo do peso, pois seu imc deu {imc:.2f}')
    if (imc >= 19) and (imc <= 24): 
        print(f'Você está com peso ideal, pois seu imc deu {imc:.2f}')
    if (imc > 25):
    	print(f'Você está acima do peso, pois seu imc deu {imc:.2f}')
else:
    if (imc < 20):
        print(f'Você está abaixo do peso, pois seu imc deu {imc:.2f}')
    if (imc > 20) and (imc <= 25): 
        print(f'Você está com peso ideal, pois seu imc deu {imc:.2f}')
    if (imc > 25) :
        print(f'Você está acima do peso, pois seu imc deu {imc:.2f}')

File: Exercícios/Exercicios/src/exercicios_java/Ex17.java
package exercicios_java;

import java.util.Scanner;

public class Ex17 {

	public static void main(String[] args) {
		// Entrar com o peso, o sexo e a altura de uma determinada pessoa. 
		// Após a digitação, exibir se esta pessoa está ou não com seu peso ideal. Fórmula: peso/altura².

		Scanner ler = new Scanner (System.in);
		
		int p, h, imc;
		String sexo;
		
		System.out.printf("Digite o seu peso: ");
		p = ler.nextInt();
		
		System.out.printf("Digite a sua altura :");
		h = ler.nextInt();
		
		imc = p/(h*h);
		
		System.out.printf("Digite o seu sexo (f/m): ");
		sexo = ler.next();
		
		if (sexo.equals('f')) {
			if (imc < 19) {
				System.out.printf("Você está abaixo do peso, pois seu imc deu %d",imc);
			} else if (imc >= 19 && imc <= 24) { 
				System.out.printf("Você está com peso ideal, pois seu imc deu %d ",imc);
			} else if (imc > 25) {
				System.out.printf("Você está acima do peso, pois seu imc deu %d", imc);	
			}
		    	
		} else {
			 if (imc < 20) { 
				 System.out.printf("Você está abaixo do peso, pois seu imc deu %d", imc);
			 }else  if (imc > 20 && imc <= 25){
				 System.out.printf("Você está com peso ideal, pois seu imc deu %d", imc);
			 } else if (imc > 25) { 
				 System.out.printf("Você está acima do peso, pois seu imc deu %d", imc); 
			 }	
		}
		
		
		
		
		
		
		
		
	}

}
